vtex-apps/login
# Feature/create pass query

#### What is the purpose of this pull request?
Add query strings `userEmail` and `flowState`

#### What problem is this solving?
When used together, they allow the login flow to begin in the create password state, with an email (with a code) sent to the user on page view. 
This depends on https://github.com/vtex/react-vtexid/pull/90.

#### How should this be manually tested?

You can test it in the following link, replacing `{yourMail}` by your email.
https://rafaprtest--storecomponents.myvtex.com/login?userEmail={yourMail}&flowState=createPass
Upon accessing the page, you will receive an email with the code that should be filled in the login page. That way, a new password can be created.

#### Types of changes

### Clubhouse hooks
[ch34488]
